packages/bridge/src/runtime/nitro/renderer.ts

```diff
@@ -1,10 +1,13 @@
+import { createServerHead } from '@unhead/vue'
+import { renderSSRHead } from '@unhead/ssr'
```

Comment:
will this work when `meta: false` is set, disabling the unhead integration?

Reply:
The same problem occurs with `meta: false` as with https://github.com/nuxt/bridge/issues/1051.
It may be easier to generate meta tags with `renderSSRHead`.

Reply:
looking at this now